File: Difficulty: Basic/Alternates in an Array/alternates-in-an-array.cpp
class Solution {
  public:
    vector<int> getAlternates(vector<int> &arr) {
        // code here
        int n=arr.size();
        vector<int>result;
        for(int i=0;i<n;i++){
            if(i%2==0){
            result.push_back(arr[i]);
            }
        }
        return result;
    }
};

File: Difficulty: Basic/Alternates in an Array/alternates-in-an-array.java
class Solution {
    // function to print alternate elements of an array
    public ArrayList<Integer> getAlternates(int arr[]) {
        // Code Here
        int n=arr.length;
        ArrayList<Integer> element= new ArrayList<>();
        
        for(int i=0;i<n;i++){
            if(i%2==0)
            element.add(arr[i]);
        }
        
        return element;
    }
}